crates/subspace-node/src/chain_spec.rs

```diff
@@ -283,14 +287,14 @@ pub fn devnet_config_compiled() -> Result<GenericChainSpec, String> {
                 enable_domains: true,
                 enable_dynamic_cost_of_storage: false,
                 enable_balance_transfers: true,
-                // TODO: Proper value here
-                confirmation_depth_k: 100,
-                // TODO: Proper value here
+                confirmation_depth_k: 5,
                 rewards_config: RewardsConfig {
                     remaining_issuance: 1_000_000_000 * AI3,
                     proposer_subsidy_points: Default::default(),
                     voter_subsidy_points: Default::default(),
                 },
+                domain_block_pruning_depth: 5,
+                staking_withdrawal_period: 5,
```

Comment:
It might also be useful to have DEV versions of the constants, so we're not repeating them a lot (and they're easier to keep in sync).

Also applies to every other place we use 5 in this PR.